src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/FilterLongReadAlignmentsSAMSpark.java

```diff
@@ -61,6 +62,12 @@
     private static final long serialVersionUID = 1L;
     private final Logger localLogger = LogManager.getLogger(FilterLongReadAlignmentsSAMSpark.class);
 
+    @Advanced
+    @Argument(doc = "parameter (float) for specifying of two configurations differ by no more than this amount, the two are considered equally good",
```

Comment:
How about, "Maximum difference between alignment configuration scores for which two configurations will be considered equally likely"?

Reply:
done